### DISCUSSION TITLE
Work to support Vulkan renderer on Android?

### DISCUSSION THREAD

**Discussion by tutacat** [ppy/osu-framework#6464] [Ideas]
Vulkan is supposed to provide higher performance with lower overhead, and Vulkan experimental renderer is supported on Linux and Windows currently.

Although work is needed for that, since the Vulkan renderer on Android is currently artifacting (background, and layers flickering) and can freeze or crash when resized. Though it seems to have slightly higher performance than OpenGL.

**Comment by smoogipoo** [comment]
We will have to move away from Veldrid (to SDL_gpu) before this is done.
